### PR TITLE
feat(schema): create TimeSeriesReadingQuantity Cost

### PR DESCRIPTION
**Pull request by macintoshpie**
TODO: After merging, ~~create new issue for removing ReadingType/Cost in 3.0~~ make a comment on issue #236 re: these changes

### CODE REVIEW

**Comment by markborkum** [comment]
.@macintoshpie The `auc:TimeSeriesReadingQuantity` element is optional, but not multi-valued. Under this proposal, within the same `auc:TimeSeries` element, would it possible to assert **both** the measured quantity and the associated cost?

Currently, in Audit Template, `auc:TimeSeries` elements (quantities) are linked to corresponding `auc:AllResourceTotal` elements (costs).

**Review by corymosiman12 — APPROVED** [review]
This seems correct to me

**Comment by corymosiman12** [comment]
@markborkum that has come up before and I think it's something we plan to tackle in a 3.0, see here: https://github.com/BuildingSync/schema/issues/236

For now, this is just intended to provide a minor fix.

**Comment by macintoshpie** [comment]
Cory is correct, this wouldn't allow for modeling cost and the measured quantity. Also thanks for bringing up the fact that we were going to consider refactoring the TimeSeries element. I'll just make a note on that issue if this gets merged

**Comment by corymosiman12** [comment]
@markborkum  - seem good?

**Review by markborkum — APPROVED** [review]
Proposal looks good, thanks.

**Review by nllong — APPROVED** [review]

File: BuildingSync.xsd [hunk 1]
```diff
@@ -4372,7 +4376,12 @@
         </xs:annotation>
         <xs:simpleType>
           <xs:restriction base="xs:string">
-            <xs:enumeration value="Currency"/>
+            <xs:enumeration value="Currency">
+              <xs:annotation>
+                <xs:documentation>WARNING: This enumeration is being deprecated, use Cost instead</xs:documentation>
+              </xs:annotation>
+            </xs:enumeration>
+            <xs:enumeration value="Cost"/>
```
Comment (nllong) [inline BuildingSync.xsd:4384]: Cost was added here. We don't want to use currency.

**Pull request merged**